script/lib/util.py

```diff
@@ -254,6 +254,8 @@ def get_platform():
 def omaha_channel(platform, arch, preview):
     if platform == 'darwin':
         if preview:
+            if release_channel() in ['nightly']:
+                return 'test-nite'
```

Comment:
I still find `nite` to be more confusing than an abbreviation of `nightly`, perhaps have it as `ntly` or `nigh` (as we do `be` for `beta`)

Reply:
I think `nite` is more natural than `ntly` or `nigh`.